src/api/recordlinker/main.py

```diff
@@ -83,13 +76,27 @@ async def not_found_handler(request, exc):
             return FileResponse(os.path.join(settings.ui_static_dir, "404.html"), status_code=404)
         raise exc
 
+    # page routes
     @app.get("/")
     @app.get("/wizard")
-    @app.get("/favicon.ico")
```

Comment:
@johanna-skylight I think we still need to serve the favicon.ico from the root directory as thats where most browsers will look for it.